src/modules/telemtry/entities/telemetry.ts

```
@@ -0,0 +1,40 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    BaseEntity,
+  } from 'typeorm';
```

ESLint

[error] Replace `⏎····Entity,⏎····PrimaryGeneratedColumn,⏎····Column,⏎····BaseEntity,⏎·` with `·Entity,·PrimaryGeneratedColumn,·Column,·BaseEntity`

(prettier/prettier)